content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -1,67 +1,236 @@
+  issued.
+- :guilabel:`Full Refund`: The credit note is auto-validated and reconciled with the invoice. *This
+  is the option to choose if a validated invoice needs to be canceled*.
+- :guilabel:`Full refund and new draft invoice`: The credit note is auto-validated and reconciled
+  with the invoice. The original invoice is duplicated as a new draft. *This is the option to choose
+  if a validated invoice needs to be modified.*
+
+.. important::
+   The :guilabel:`Credit Method` options will *not* be available for invoices that have already been
+   paid.
+
+Make any necessary changes to the details of the credit note and click :guilabel:`Reverse.` Then
+click :guilabel:`Confirm` to post the credit note.
+
+.. image:: after_sales/after-sales-draft-credit.png
+   :align: center
+   :alt: View of a draft credit note.
+
+Once the credit note has been posted, a :guilabel:`Credit Notes` smart button will be added to the
+:menuselection:`Helpdesk` :guilabel:`Ticket`.
+
+.. image:: after_sales/after-sales-credit-note-smart-button.png
+   :align: center
+   :alt: View of smart buttons on a ticket focusing on the credit note button.
+
+.. seealso::
+   - :doc:`/applications/finance/accounting/receivables/customer_invoices/credit_notes`
+
+Generate coupons from a ticket
+==============================
+
+Coupons can be used to alter the price of products or orders. The usage constraints of a coupon are
+defined by conditional rules. *Coupon Programs* are configured in the :menuselection:`Sales` or
+:menuselection:`Website` applications.
+
+.. note::
+   The :menuselection:`eCommerce` module must be installed in order to create coupon codes from the
+   :menuselection:`Website` application.
+
+.. image:: after_sales/after-sales-coupon-setup.png
+   :align: center
+   :alt: View of a coupon program configuration page.
+
+To generate a coupon, open a :menuselection:`Helpdesk` :guilabel:`Ticket` and click on the
+:guilabel:`Coupon` button in the upper left corner. Select an option from the :guilabel:`Coupon
+Program` drop-down menu. Click :guilabel:`Generate`.
+
+.. image:: after_sales/after-sales-generate-coupon.png
+   :align: center
+   :alt: View of a coupon generation window.
+
+The :guilabel:`Coupon Code` can be copied directly from the pop-up window (by clicking the
+:guilabel:`Copy` button), or sent in an email by clicking :guilabel:`Send`.
+
+.. note::
+   All followers of the :guilabel:`Ticket` will be added as recipients to the email. Additional
+   recipients can be added to the email as well.
+
+   .. image:: after_sales/after-sales-coupon-email.png
+      :align: center
+      :alt: View of an email draft window with coupon code.
+
+Once a :guilabel:`Coupon Code` has been generated, a :guilabel:`Coupons` smart button will be added
+to the top of the :menuselection:`Helpdesk` :guilabel:`Ticket`. Click here to view the coupon code,
+expiration date, and additional information.
```

Comment:
check tag formatting in this entire paragraph
can we reword this as a clearer instruction to "click X" instead of implied "click here"?